src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -700,14 +700,15 @@ def test_process_declarations_accessibility():
     reader = FortranStringReader("private :: x\n"
                                  "real :: x\n")
     fparser2spec = Specification_Part(reader).content
-    processor.process_declarations(sched, fparser2spec, [])
+    _, vis_map = processor.process_access_statements(fparser2spec)
+    processor.process_declarations(sched, fparser2spec, [], vis_map)
```

Comment:
It seems the test docstring is outdated as mappings are provided in 2 out of the 3 process_declaration calls. Can you update the docstring.

Reply:
Oops, I got a bit gung-ho when making all the tests work.